src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -246,6 +246,7 @@ module AssemblyConstants =
     let ToffoliSimulator = "ToffoliSimulator"
     let ResourcesEstimator = "ResourcesEstimator"
     let ExposeReferencesViaTestNames = "ExposeReferencesViaTestNames"
+    let ConcreteIntrinsicGen = "ConcreteIntrinsicGen"
```

Comment:
I have a slight preference for using the active verb at the front, similar to `ExposeReferencesViaTestName`, so I think I'd like `GenerateConcreteIntrinsic` instead of `ConcreteIntrinsicGen`. I feel like that more clearly underlines the behavior that will change when the property is set to true.

Reply:
Changed to use GenerateConcreteIntrinsic